src/components/PaintControls.vue

```diff
@@ -134,10 +134,13 @@ export default defineComponent({
     });
 
     const setBrushColor = (color: string) => {
-      const hexa = `${color}FF`.toUpperCase();
-      if (hexa in hexToValue) {
-        paintStore.setBrushValue(hexToValue[hexa]);
+      const withAlpha = color.length === 9 ? color : `${color}FF`;
```

Comment:
Nit: I'd recommend adding a comment explaining why the number 9 shows up here or name this constant.